boards/arm/a1x/pcduino-a10/scripts/Make.defs

```diff
@@ -35,8 +35,8 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
 endif
 
 ARCHCPUFLAGS = -mcpu=cortex-a8 -mfpu=vfpv4-d16
-ARCHCFLAGS = -fno-builtin
-ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new -fno-rtti
+ARCHCFLAGS = -fno-common -fno-builtin
+ARCHCXXFLAGS = -fno-common -fno-builtin -fno-exceptions -fcheck-new -fno-rtti
```

Comment:
I think we should migrate this to `ARCHCXXFLAGS = $(ARCHCFLAGS) ...` in the next PR if possible

Reply:
Yes, I think so.